extensions/sql-database-projects/src/models/project.ts

```diff
@@ -567,19 +580,33 @@ export class ProjectEntry {
 /**
  * Represents a database reference entry in a project file
  */
-export class DatabaseReferenceProjectEntry extends ProjectEntry {
-	constructor(uri: Uri, public databaseLocation: DatabaseReferenceLocation, public name?: string) {
+
+export interface IDatabaseReferenceProjectEntry extends ProjectEntry {
+	databaseReferenceType: DatabaseReferenceType;
+	databaseName: string;
+	sqlCmdName?: string | undefined;
+}
+
+export class DacpacReferenceProjectEntry extends ProjectEntry implements IDatabaseReferenceProjectEntry {
```

Comment:
Since this class implements IDatabaseReferenceProjectEntry, which already extends ProjectEntry- is there a need to specifically have this class extend ProjectEntry as well?

Reply:
it's needed to call the super constructor